File: POTW/week-6/CS-Solution6.py
# Krish Shah
# Determine the number of people with COVID after x amount of days.

initial_infected = int(input())
infection_rate = int(input())
num_days = int(input())

num_infected = initial_infected
for i in range(1, num_days+1):
    num_infected += initial_infected * (infection_rate ** i)

print(num_infected)

File: POTW/week-6/CS_Solution6.java
/**
 * Krish Shah
 * Determine the number of people with COVID after x amount of days.
 */

package potw;

import java.util.Scanner;

public class CS_Solution6 {

	public static int findNumInfected(int initialInfected, int infectionRate, int numDays) {
		int numInfected = initialInfected;
		int dailyInfectionRate = infectionRate;
		for (int i = 1; i <= numDays; i++) {
			numInfected += dailyInfectionRate;
			dailyInfectionRate *= infectionRate;
		}
		return numInfected;
	}

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int initialInfected = scanner.nextInt();
		int infectionRate = scanner.nextInt();
		int numDays = scanner.nextInt();

		System.out.println(findNumInfected(initialInfected, infectionRate, numDays));
	}

}